src/main/java/de/tum/in/www1/artemis/domain/TempIdObject.java

```diff
@@ -1,33 +1,46 @@
 package de.tum.in.www1.artemis.domain;
 
+import java.io.Serializable;
 import java.util.Objects;
 
 import jakarta.persistence.Transient;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonView;
+
+import de.tum.in.www1.artemis.domain.view.QuizView;
 
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public abstract class TempIdObject extends DomainObject {
+public abstract class TempIdObject implements Serializable {
```

Comment:
Why do you no longer extend DomaintObject? Does this have an effect on other classes that extend the TempIdObject?